pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/LLCRealtimeClusterSplitCommitIntegrationTest.java

```diff
@@ -20,7 +20,14 @@
 
 import org.apache.commons.configuration.Configuration;
 import org.apache.pinot.common.utils.CommonConstants;
+import org.apache.pinot.common.utils.StringUtil;
```

Comment:
Looks like there are no change to this file. Can we revert this file to the original please?